### PR TITLE
Feature/heic to jpeg

### PR DESCRIPTION
**Pull request by mneuwert**
## Description
Added possibility to convert photos stored in HEIC format to JPEG before uploading. Also added corresponding settings.

## Motivation and Context
Many users who would like to open their photos on platforms other than macOS / iOS where the HEIF support is either limited or non-existing.

## Known Issues
Metadata is not preserved when the photo is uploaded but it seems to apply also to non-converted photos. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ownCloud/Settings/PhotoUploadSection.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//
+//  PhotoUploadSection.swift
+//  ownCloud
+//
+//  Created by Michael Neuwert on 25.04.2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
```
Comment (hosy) [inline ownCloud/Settings/PhotoUploadSection.swift]: Only license header is missing.

**Comment by hosy** [comment]
@mneuwert fast and nice implementation. Thank you! Only a license header is missing.

File: ownCloud/Settings/PhotoUploadSection.swift (hunk 1 above)
Comment (mneuwert) [reply ownCloud/Settings/PhotoUploadSection.swift]: @hosy thanks, fix pushed

**Comment by jesmrec** [comment]
Approved

**Pull request merged**